patches/util-linux/0001-Stub-out-several-unsupported-things-on-managarm.patch

```diff
@@ -0,0 +1,175 @@
+From 3b0ef9e87835d779a439a00088907d2e6e71feca Mon Sep 17 00:00:00 2001
+From: Dennis Bonke <admin@dennisbonke.com>
+Date: Thu, 21 Jan 2021 12:23:48 +0100
+Subject: [PATCH 1/2] Stub out several unsupported things on managarm
+
+Signed-off-by: Dennis Bonke <admin@dennisbonke.com>
+---
+ disk-utils/cfdisk.c           | 1 +
+ include/namespace.h           | 2 ++
+ include/path.h                | 2 ++
+ lib/cpuset.c                  | 2 ++
+ lib/path.c                    | 2 ++
+ libblkid/src/topology/ioctl.c | 2 ++
+ login-utils/logindefs.c       | 2 +-
+ login-utils/lslogins.c        | 2 +-
+ misc-utils/lsblk.h            | 2 ++
+ term-utils/agetty.c           | 8 ++++++++
+ 10 files changed, 23 insertions(+), 2 deletions(-)
+
+diff --git a/disk-utils/cfdisk.c b/disk-utils/cfdisk.c
+index d96b6e9e0..3a5b20939 100644
+--- a/disk-utils/cfdisk.c
++++ b/disk-utils/cfdisk.c
+@@ -22,6 +22,7 @@
+ #include <libsmartcols.h>
+ #include <sys/ioctl.h>
+ #include <libfdisk.h>
++#include <termios.h>
+ 
+ #ifdef HAVE_LIBMOUNT
+ # include <libmount.h>	/* keep it optional for non-linux systems */
+diff --git a/include/namespace.h b/include/namespace.h
+index 2d0a56e02..75d0f439d 100644
+--- a/include/namespace.h
++++ b/include/namespace.h
+@@ -7,6 +7,7 @@
+  */
+ #ifndef UTIL_LINUX_NAMESPACE_H
+ # define UTIL_LINUX_NAMESPACE_H
++#ifndef __managarm__
```

Comment:
What do we patch out here? (And in other places?)

Reply:
Several linux only features like namespaces or the Linux specific cpu interface